updating/kmm-preflight-validation.adoc

```diff
@@ -0,0 +1,22 @@
+:_content-type: ASSEMBLY
+[id="kmm-preflight-validation"]
+= Preflight validation for Kernel Module Management (KMM) Modules
+include::_attributes/common-attributes.adoc[]
+:context: kmm-preflight-validation
+
+toc::[]
+
+Before performing an upgrade on the cluster with applied KMM modules, the administrator needs to verify
+that installed kernel modules (via KMM) are able to be installed on the nodes after the cluster upgrade and
```

Comment:
See IBM SG for word usage on "via" and avoiding using parentheses in general text.

Reply:
Changed to "kernel modules installed using KMM are..."